exercises/change/description.md

```diff
@@ -1,3 +1,5 @@
+Correctly determine change to be given using the least number of coins
+
```

Comment:
Missing dot at the end of the line. Also duplicate description below, although the description below does give more information.